static/app/views/settings/organizationIntegrations/abstractIntegrationDetailedView.tsx

```diff
@@ -19,7 +19,7 @@ import RequestIntegrationButton from './integrationRequest/RequestIntegrationBut
 
 export type Tab = 'overview' | 'configurations' | 'features';
 
-interface AlertType extends AlertProps {
+export interface AlertType extends AlertProps {
```

Comment:
both of these can be moved to `integrationLayout.tsx` after completing this conversion